src/main/java/com/documaster/rms/noark/ws/noarkentities/NasjonalIdentifikator.java

```diff
@@ -0,0 +1,70 @@
+package com.documaster.rms.noark.ws.noarkentities;
+
+import com.documaster.rms.noark.ws.client.action.LinkAction;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public abstract class NasjonalIdentifikator<TEntity extends NasjonalIdentifikator<TEntity>> extends NoarkEntityBase<TEntity> {
```

Comment:
Line is a bit longer.

Reply:
Addressed with [this commit](https://github.com/documaster/ws-client-n5-java/pull/16/commits/b3ed4002853a3f25c3ff0b722ef0640c6eeea787).